package.json

```diff
@@ -105,5 +106,8 @@
     "react-dom": "^17.0.2",
     "stylelint": "^14.3.0",
     "typescript": "^4.5.5"
+  },
+  "peerDependencies": {
+    "@yandex-cloud/browserslist-config": "^1.0.1"
```

Comment:
Questionable.
I added `@yandex-cloud/browserslist-config` in `ydb-ui-components` peerDependencies because browserslist in package.json of `ydb-ui-components` extends a config from `@yandex-cloud/browserslist-config`. This causes build of `ydb-embedded-ui` to crash because of a missing dependency. Adding the package to peer deps doesn't solve the problem, but provides a hint to install this package.

But I assume that build of any project that depends on `ydb-embedded-ui` will similarly fail for the same reason. And I don't like the solution with specifying this peer dependency. It's kinda unhelpful.

I guess we need to solve the issue with crashing build some other way.